applications/crossbar/priv/api/swagger.json

```diff
@@ -8242,16 +8242,66 @@
                     "description": "kazoo_couch allow maintenance db delete",
                     "type": "boolean"
                 },
+                "autocompaction_check": {
+                    "default": 60000,
+                    "description": "kazoo_couch autocompaction check",
+                    "type": "integer"
+                },
+                "compact_automatically": {
+                    "default": true,
+                    "description": "kazoo_couch compact automatically",
+                    "type": "boolean"
+                },
                 "default_chunk_size": {
                     "default": 1000,
                     "description": "kazoo_couch default chunk size",
                     "type": "integer"
                 },
+                "max_compacting_shards": {
+                    "default": 2,
+                    "description": "kazoo_couch max compacting shards",
+                    "type": "integer"
+                },
+                "max_compacting_views": {
+                    "default": 2,
+                    "description": "kazoo_couch max compacting views",
+                    "type": "integer"
+                },
                 "max_concurrent_docs_to_archive": {
                     "default": 500,
                     "description": "kazoo_couch max concurrent docs to archive",
                     "type": "integer"
                 },
+                "max_wait_for_compaction_pids": {
+                    "default": 360000,
+                    "description": "kazoo_couch max wait for compaction pids",
+                    "type": "integer"
+                },
+                "min_data_size": {
+                    "default": 131072,
+                    "description": "kazoo_couch min data size",
+                    "type": "integer"
+                },
+                "min_ratio": {
+                    "default": 1.2,
+                    "description": "kazoo_couch min ratio",
+                    "type": "number"
+                },
+                "sleep_between_compaction": {
+                    "default": 60000,
+                    "description": "kazoo_couch sleep between compaction",
+                    "type": "integer"
+                },
+                "sleep_between_poll": {
+                    "default": 3000,
+                    "description": "kazoo_couch sleep between poll",
+                    "type": "integer"
+                },
+                "sleep_between_views": {
```

Comment:
Could the flag mention even the abbreviated time unit? d, s, ms, ...
For Boolean values staring the flag with "should" is also nice